src/ctia/schemas/graphql/helpers.clj

```diff
@@ -289,11 +308,14 @@
   [^String object-name
    ^String description
    fields]
-  (-> (GraphQLInputObjectType/newInputObject)
-      (.name object-name)
-      (.description description)
-      (add-input-fields fields)
-      .build))
+    (get-or-update-type-registry
```

Comment:
Please temporarily permit the funny indentation, a forthcoming commit with wrap this entire return value in a function and restore some order.